### PR TITLE
Added procedures for new stats

### LINKED ISSUE #25: Extension mechanism needs to be clear
For WebRTC-1.0, stats needs to be a stable reference (with all the stats that are in the MTI set).

Adding stats needs to be a process that's clear to people who want to do it.

### PR DESCRIPTION
**Pull request by alvestrand**
The approach sketched is "living document".

Fixes #25

### CODE REVIEW

**Comment by alvestrand** [comment]
@fluffy @dontcallmedom @juberti please comment on approach

File: webrtc-stats.html [hunk 1]
```diff
@@ -88,6 +88,28 @@
...
     </section><!-- VS: getStats() remains in base spec! -->
+    <section>
+      <h2>Maintenance procedures for stats types</h2>
+      <p>This document, in its editors' draft form, serves as the repository
+        for the currenly defined set of stats types.
+      </p>
+      <p>
+        If a need for a new stats dictionary or dictionary member is found,
+        an issue should be raised against this spec on Github, and the W3C
+        consensus process will decide on whether the stat should be added
```
Comment (fluffy) [inline webrtc-stats.html]: What is the W3C consensus process for a closed WG ?

**Comment by fluffy** [comment]
Needless to say, I would prefer a process where an someone could get a stat name and start implementing that in parallel with it becoming a standards and know that there would be no name conflicts.

**Comment by vr000m** [comment]
The process for getting a stat name for a well documented metric should indeed be documented.

**Comment by alvestrand** [comment]
The experience I have with adding stats so far is that there are two controversial things:
- The name
- The definition (usually the units, sometimes how the value is computed)
Declaring either to be frozen by the time we get a proposal seems not likely to be acceptable.

**Comment by vr000m** [comment]
The bar is set as: Expert review, similar to IANA process

+ citable reference
+ a good name

there was more content on how the actual process might work. need input from @fluffy

**Comment by alvestrand** [comment]
Current thought: Add "bar is at Expert Review" and "after WG close, an expert will have to be appointed", and merge.

**Pull request merged**